qemu/tests/timerdevice_boot.py

```diff
@@ -58,6 +59,21 @@ def get_hwtime(session):
             raise err
         return guest_time
 
+    def verify_time_hostsys_guesthw(session):
+        """
+        Verify host's system time with guest's hardware time.
+        """
+        error_context.context("Check the hardware time on guest", logging.info)
+        host_systime, guest_systime = utils_test.get_time(session, time_command,
```

Comment:
`guest_systime` Unused?

Reply:
If so, I would suggest have a parameter to control comparing to guest system time or guest hardware time, this way you can include the implementation in L#163 in this function too.